tests/baselines/reference/jsFileCompilationExternalPackageError.errors.txt

```diff
@@ -0,0 +1,19 @@
+tests/cases/compiler/moduleA/a.js(2,17): error TS2656: Exported external package typings file 'tests/cases/compiler/node_modules/b.ts' is not a module. Please contact the package author to update the package definition.
+
+
+==== tests/cases/compiler/moduleA/a.js (1 errors) ====
+    
+    import {a} from "b";
+                    ~~~
+!!! error TS2656: Exported external package typings file 'b.ts' is not a module. Please contact the package author to update the package definition.

```

Comment:
i guess i do not understand what is an error a user would expect from a JS file and what is not..